src/formatDate.js

```diff
@@ -50,7 +50,60 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let day = 0;
+  let month = 0;
+  let year = 0;
```

Comment:
```suggestion
  let day;
  let month;
  let year;
```
These initial values have no use